scene/gui/color_picker.cpp

```diff
@@ -158,24 +158,28 @@ void ColorPicker::_notification(int p_what) {
 				return;
 			}
 			DisplayServer *ds = DisplayServer::get_singleton();
-			Vector2 ofs = ds->mouse_get_position();
-			picker_window->set_position(ofs - Vector2(28, 28));
```

Comment:
The only relevant change is removal of this line, which is redundant, but should not have any significant effect (window position reset later, but I guess can cause some flicker).

Reply:
I agree, the other changes are more linting than fixes.